src/docs/arch-compact.dox

```diff
@@ -81,7 +81,8 @@ Background compaction retrieves the files to compact by walking the metadata fil
 background compaction reduces the number of checkpoints by skipping the first one described in the
 section above. Furthermore, background compaction gathers statistics over time to keep track of tables
 where compaction is successful. Using those statistics, the service focuses on tables where compaction
-is likely to reclaim space and avoids unnecessary attempts.
+is likely to reclaim space and avoids unnecessary attempts. Finally, if \c eviction_dirty_trigger has been reached,
```

Comment:
This doesn't mention the second reason why background compaction pauses: the cache is almost full.

Reply:
I missed it, see 8b097254fff78dae84013b9d4846d84f31e4423b.